chord_metadata_service/experiments/tests/test_api.py

```diff
@@ -0,0 +1,133 @@
+import os
+from uuid import uuid4
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+from chord_metadata_service.chord.models import Project, Dataset, TableOwnership, Table
+from chord_metadata_service.chord.tests.constants import VALID_DATA_USE_1
+from chord_metadata_service.chord.data_types import DATA_TYPE_PHENOPACKET, DATA_TYPE_EXPERIMENT
+from chord_metadata_service.chord.ingest import (
+    WORKFLOW_INGEST_FUNCTION_MAP,
+    WORKFLOW_PHENOPACKETS_JSON,
+    WORKFLOW_EXPERIMENTS_JSON,
+)
+
+
+EXAMPLE_INGEST_OUTPUTS_PHENOPACKETS_JSON = {
+    "json_document": os.path.join(os.path.dirname(__file__), "example_phenopackets.json"),
+}
+
+EXAMPLE_INGEST_OUTPUTS_EXPERIMENTS_JSON = {
+    "json_document": os.path.join(os.path.dirname(__file__), "example_experiments.json"),
+}
+
+
+class GetExperimentsAppApisTest(APITestCase):
```

Comment:
Very minor: I don't understand what "App" means in this context (i.e. "ExperimentsApp", "Experiments app APIs")
I would think it's question of experiment results in this test, but I may miss something lexic wise?

Reply:
I am partially replying to myself: these tests are not only about experimental results, but also experiments.